### ISSUE TITLE
User Story:  Provision and Setup Front-End Development Environnment

### ISSUE THREAD

**Issue by pdxdiver** [hackoregon/hack-university-data-architecture#3]
As a front-end developer, I want to be able to be up and running (full working environment) in less than 1 hour

**Comment by meganmckissack** [comment]
Do we have a rough eta on when we'll have a react boilerplate?

**Comment by scottslewis** [comment]
By boilerplate do you mean a virtual box instance with react tools (like we did for datascience tools?).   If so then we should coordinate this with David Daniel, as he's decided that a 'toolchain for ui/front-end/web development' will be his Data Architecture course project.   We should also speak with Nim and Ryan, as I expect they and others from the UI design course will have thoughts about what this should (minimally to start) consist of.   Please remind me Wed evening and we will speak with David about it (and/or feel free to slack David directly).

**Comment by meganmckissack** [comment]
Yes, similar.  I'll connect with David about that.  The product design class only covers mostly design aspects of the projects and their last class is tonight but I have a team of front end leads that can do some react package install tests.
